contracts/interfaces/IFractalUsul.sol

```diff
@@ -2,44 +2,166 @@
 pragma solidity ^0.8.0;
 
 import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+import "../FractalUsul.sol";
 
 interface IFractalUsul {
-    /// @dev Returns the hash of a transaction in a proposal.
-    /// @param proposalId the proposal to inspect.
-    /// @param index the transaction to inspect.
-    /// @return transaction hash.
-    function getTxHash(uint256 proposalId, uint256 index)
-        external
-        view
-        returns (bytes32);
+    enum ProposalState {
+        Active,
+        Canceled,
+        Timelocked,
+        Executed,
+        Executable,
+        Uninitialized
+    }
+
+    /// @notice Enables a voting strategy that can vote on proposals, only callable by the owner
+    /// @param strategy Address of the strategy to be enabled
+    function enableStrategy(address strategy) external;
+
+    /// @notice Disables a voting strategy on the module, only callable by the owner
+    /// @param prevStrategy Strategy that pointed to the strategy to be removed in the linked list
+    /// @param strategy Strategy to be removed
+    function disableStrategy(address prevStrategy, address strategy) external;
+
+    /// @notice This method submits a proposal which includes metadata strings to describe the proposal
+    /// @param strategy Address of Voting Strategy, under which proposal submitted
+    /// @param data - any additional data, which would be passed into IStrategy.receiveProposal
+    /// @param transactions - array of transactions to execute
+    /// @param title - proposal title, emitted in ProposalCreated
+    /// @param description - proposal description, emitted in ProposalCreated
+    /// @param documentationUrl - proposal documentation/discussion URL, emitted in ProposalCreated
+    function submitProposal(
+        address strategy,
+        bytes memory data,
+        FractalUsul.Transaction[] calldata transactions,
+        string calldata title,
+        string calldata description,
+        string calldata documentationUrl
+    ) external;
+
+    /// @notice Cancels an array of proposals
+    /// @param proposalIds Array of proposals to cancel
+    function cancelProposals(uint256[] memory proposalIds) external;
+
+    /// @notice Called by the strategy contract when the proposal vote has succeeded
+    /// @param proposalId the identifier of the proposal
+    /// @param timeLockPeriod the optional delay time
+    function timelockProposal(uint256 proposalId, uint256 timeLockPeriod) external;
```

Comment:
another instance of timeLockPeriod that should be lowercase L

Reply:
Fixed this